tests/Feature/PostTest.php

```diff
@@ -0,0 +1,73 @@
+<?php
+
+namespace Tests\Feature;
+
+use Illuminate\Foundation\Testing\RefreshDatabase;
+use Illuminate\Foundation\Testing\WithFaker;
+use Tests\TestCase;
+use App\Post;
+use Illuminate\Support\Facades\Storage;
+use Illuminate\Support\Str;
+use Image;
+
+class PostTest extends TestCase
```

Comment:
Good attempt with Tests. Just keep in mind that we want to test that your implementation works (see HTTP tests) versus testing the framework features that are covered by their own test suite.

Reply:
Just need to now add in HTTP tests and will put forward a new pull request.